src/liveEditor/utils/multipleElementAddButton.ts

```diff
@@ -138,7 +138,7 @@ export function handleAddButtonsForMultiple(
         );
 
         if (!fieldDisabled) {
-            const previousButton = generateAddInstanceButton(() => {
+            const previousButtonWrapper = generateAddInstanceButton(() => {
                 liveEditorPostMessage?.send(
```

Comment:
The `generateAddInstanceButton` should return a button to reduce complexity in the code.